app/src/debug/AndroidManifest.xml

```diff
@@ -36,6 +36,7 @@
         android:protectionLevel="signature"/>
     <uses-permission android:name="${applicationId}.permission.RECEIVE_ADM_MESSAGE" />
     <uses-permission android:name="com.amazon.device.messaging.permission.RECEIVE" />
+    <uses-permission android:name="android.permission.WAKE_LOCK"/>
```

Comment:
nit: can we add a comment mentioning this is included by the dependency? You can run a release build and check `app/build/outputs/logs/manifest-merger-system-release-report.txt` to see which dependency added it and include the name of it in the comment.

I think the comment will be helpful, just in case anyone tries to remove the permission and expects for it to go away for app users.